src/server/rest/v1/service/ChargingStationService.ts

```diff
@@ -56,230 +57,39 @@ import UtilsService from './UtilsService';
 const MODULE_NAME = 'ChargingStationService';
 
 export default class ChargingStationService {
+
   public static async handleUpdateChargingStationParams(action: ServerAction, req: Request, res: Response, next: NextFunction): Promise<void> {
     // Filter
     const filteredRequest = ChargingStationValidatorRest.getInstance().validateChargingStationParametersUpdateReq({ ...req.params, ...req.body });
     // Check the Charging Station
     const chargingStation = await UtilsService.checkAndGetChargingStationAuthorization(
-      req.tenant, req.user, filteredRequest.id, Action.UPDATE, action, null, { withSiteArea: true });
-    // OCPI Charging Station
-    if (!chargingStation.issuer) {
-      throw new AppError({
-        errorCode: HTTPError.GENERAL_ERROR,
-        message: `Charging Station '${chargingStation.id}' not issued by the organization`,
-        module: MODULE_NAME, method: 'handleUpdateChargingStationParams',
-        user: req.user,
-        action
-      });
-    }
-    // Site Area
+      req.tenant, req.user, filteredRequest.id, Action.UPDATE, action, null, { issuer: true, withSiteArea: true });
+    // Check and get site area authorizations
     let siteArea: SiteArea = null;
     if (Utils.isComponentActiveFromToken(req.user, TenantComponents.ORGANIZATION) && filteredRequest.siteAreaID) {
       siteArea = await UtilsService.checkAndGetSiteAreaAuthorization(
-        req.tenant, req.user, filteredRequest.siteAreaID, Action.READ, action, null, { withSite: true });
-    }
-    // Check Auth
-    if (!await Authorizations.canUpdateChargingStation(req.user, siteArea ? siteArea.siteID : null)) {
-      throw new AppAuthError({
-        ...LoggingHelper.getChargingStationProperties(chargingStation),
-        errorCode: HTTPAuthError.FORBIDDEN,
-        user: req.user,
-        action: Action.UPDATE, entity: Entity.CHARGING_STATION,
-        module: MODULE_NAME, method: 'handleUpdateChargingStationParams',
-        value: chargingStation.id,
-      });
+        req.tenant, req.user, filteredRequest.siteAreaID, Action.READ, action, null, { issuer: true, withSite: true });
```

Comment:
Same remark on 'issuer' flag, should be in auth def.
Double check all issuer calls in this file.